benchmarks/railsbench/benchmark.rb

```diff
@@ -29,6 +29,7 @@
     unless response_array.first == 200
       raise "HTTP response is #{response_array.first} instead of 200. Is the benchmark app properly set up? See README.md."
     end
+    response_array.last.close # Response might be a Rack::BodyProxy and MUST be closed.
```

Comment:
question: Do you mean the rest of the elements in the array may not be `Rack::BodyProxy` and not need to be closed?

Reply:
The `response_array` is `[status, headers, body]`, only body need to be closed.

Reply:
Ah, got it. I thought it's an array of responses but it's just a single response. That makes sense.

Reply:
The comment says `might be`.  Could it be anything else?

Reply:
Yeah, could be an IO, but also something that doesn't respond to `close` at all. But in this case it's always true. If one day it's not, we can do a `respond_to?`.

https://github.com/rack/rack/blob/main/SPEC.rdoc#the-body-

> The Body must either be consumed or returned. The Body is consumed by optionally calling either each or call. Then, if the Body responds to close, it must be called to release any resources associated with the generation of the body. In other words, close must always be called at least once;